### DISCUSSION TITLE
Schalter im Node-Red?

### DISCUSSION THREAD

**Discussion by wolliboy** [camueller/SmartApplianceEnabler#190] [Installation und Betrieb]
Moin.
Wofür ist der Schalter im Node-Red Dashboard?
![Screenshot_20220207-092058_Chrome](https://user-images.githubusercontent.com/98582148/152750908-ed5a4e0d-9608-47c9-b7d9-02a5f47805c0.jpg)

Grüße

**Comment by camueller** [comment]
Dieser Schalter signalisiert den aktuellen Zustand des Schalters bzw. der Wallbox. Der Screenshot zeigt "eingeschaltet", auch wenn die aktuellen Werte für Leistungsaufnahme und bisherige Energiemenge daneben nicht so sinnvoll aussehen. Bei mir stehen dort sinnvolle Werte weshalb ich davon ausgehe, dass etwas mit der Konfiguration Deine go-eChargers nicht stimmt.

**Reply by wolliboy** [reply]
Ich habe die Wallbox nicht als Wallbox konfiguriert, sondern als Sonstige und immer eingeschaltet. Dann passt das ja. 

Die Steuerung über den SHM möchte ich nicht nutzen, es soll nur der Energieverbrauch der Wallbox im Sunny Portal angezeigt werden und das funktioniert ja.

**Comment by camueller** [comment]
Die neben dem Schalter angzeigten Werte sind die, welche auch an den SHM übermittel werden. Insofern wundert es mich, wenn im Portal etwas Sinnvolles ankommt, wenn bei Dir "0 W" angezeigt wird.

**Reply by wolliboy** [reply]
In dem Zeitpunkt hat er auch nicht geladen, also passen die 0W. Warum da allerdings -0,77kWh stehen...?
